enterprise/__init__.py

```diff
@@ -4,6 +4,6 @@
 
 from __future__ import absolute_import, unicode_literals
 
-__version__ = '0.1.2'
+__version__ = '0.1.3'
```

Comment:
@zubair-arbi an ad-hoc one - increase PATCH with every PR and increase MINOR whenever someone feels it is about time. And increase MAJOR to 1 when it is officially released (and I'm not aware if we ever increased anything beyond 1.x). Might be a good occasion to revise that.

@mattdrayer @haikuginger @saleem-latif @asadiqbal08  Thoughts?